sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -781,4 +782,3 @@ class AdaptiveQueryExecSuite
     }
   }
 }
-
```

Comment:
IIRC, we always need a new line at the end of a file?

Reply:
We still have.